lumicks/pylake/kymotracker/tests/test_refinement.py

```diff
@@ -158,6 +158,24 @@ def test_gaussian_refinement_fixed_background(kymogroups_2lines, fit_mode):
     )
 
 
+def test_gaussian_proximity(kymogroups_close_lines):
```

Comment:
maybe `test_gaussian_refinement_overlap()`? for consistency with other test names?

Reply:
👍